lib/index.js

```diff
@@ -74,17 +74,10 @@ export default {
 
         // Call ERB to "de-templatize" the code
         return helpers.exec(this.erbPath, erbArgs,
-          { stdin: text, cwd: fileDir }
+          { stdin: text.replace(/<%=/g, '<%'), cwd: fileDir }
```

Comment:
I'm not sure, I'll refer back to the "I don't use Ruby" as to why :stuck_out_tongue:

I just know that this package claims to understand `text.html.erb` and `text.html.ruby` scopes, and previously it would only modify the output if the file was marked as the `text.html.ruby` scope.